WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinDetailsViewModel.cs

```diff
@@ -29,8 +29,7 @@ public CoinJoinDetailsViewModel(UiContext uiContext, CoinJoinHistoryItemViewMode
 
 		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
 		NextCommand = CancelCommand;
-
-		ConfirmationTime = TimeSpan.Zero; // TODO: Calculate confirmation time
+		ConfirmationTime = coinJoin.CoinJoinTransaction.TryGetConfirmationTime(out var estimation) ? estimation : null;
```

Comment:
Previously for no estimation it was zero. Now it's null. I think null is better, but did you test the UI that it doesn't fail?

Reply:
Yes, I've tested. When the estimation is null, the section in the GUI is hidden and nothing unexpected happens.